tests/Backend/CommonPart1/CreateTableWithConfigurationTest.php

```diff
@@ -0,0 +1,344 @@
+<?php
+
+namespace Keboola\Test\Backend\CommonPart1;
+
+use Keboola\StorageApi\Client;
+use Keboola\StorageApi\ClientException;
+use Keboola\StorageApi\Components;
+use Keboola\StorageApi\Options\Components\Configuration;
+use Keboola\StorageApi\Options\TableWithConfigurationOptions;
+use Keboola\Test\ClientProvider\ClientProvider;
+use Keboola\Test\StorageApiTestCase;
+
+class CreateTableWithConfigurationTest extends StorageApiTestCase
+{
+    public const COMPONENT_ID = 'keboola.app-custom-query-manager';
+
+    private ClientProvider $clientProvider;
+    private Client $client;
+    private Components $componentsClient;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        // check feature
+        $token = $this->_client->verifyToken();
+        if (!in_array('tables-with-configuration', $token['owner']['features'])) {
+            $this->markTestSkipped(sprintf('Creating tables from configurations feature is not enabled for project "%s"', $token['owner']['id']));
+        }
+
+        // init buckets
+        $this->initEmptyTestBucketsForParallelTests();
+
+        $this->clientProvider = new ClientProvider($this);
+        $this->client = $this->clientProvider->createClientForCurrentTest();
+
+        // init configurations
+        try {
+            $this->cleanupConfigurations();
```

Comment:
Neměly by se mazat jen konfigurace ke konkrétní tabulce? Resp. jen konfigurace tabulek? Takhle to není moc parallel friendly, když to smaže všechny.

Reply:
Můžeš si udělat konfiguraci s parallel-friendly-id a tu si pomazat přímo v testu... případně tady, ale mazat jen ty konkrétní a ne všechny